src/components/side_panel/find_and_replace/find_and_replace.ts

```diff
@@ -92,7 +100,6 @@ export class FindAndReplacePanel extends Component<Props, SpreadsheetChildEnv> {
   }
 
   onFocusSidePanel() {
-    this.state.searchOptions.searchFormulas = this.env.model.getters.shouldShowFormulas();
```

Comment:
since we just approved #2680, you should be able to get rid of `onFocusSidePanel`altogether